core/src/main/java/com/google/cloud/sql/core/CoreSocketFactory.java

```diff
@@ -341,23 +349,31 @@ Socket createSslSocket(String instanceName, List<String> ipTypes, AuthType authT
     }
   }
 
+  @VisibleForTesting
```

Comment:
What would it take to test through the public interface? Ditto below.

Reply:
This method is only ever used in the test cases. I will update the test cases instead and get rid of this method.